pymodbus/file_message.py

```diff
@@ -128,7 +129,16 @@ def execute(self, context):
         # is too big, return an error.
         files = []
         return ReadFileRecordResponse(files)
+    
+    def get_response_pdu_size(self):
+        """
+        Func_code (1 byte) + Byte Count(1 byte) + Byte Count2 (1 byte) + Reference Type (1 byte) + registers to read for file
+        :return: 
+        """
+        for record in self.records:
+            self.count+=record.record_length
 
+        return 4 + self.count
```

Comment:
This needs to change per https://modbus.org/docs/Modbus_Application_Protocol_V1_1b3.pdf.